test_scripts/API/VehicleData/Refactor_Fuel_Information/006_OnVehicleData_with_only_one_param_in_fuelRange_structure.lua

```diff
@@ -0,0 +1,38 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@GetmanetsIrina , please add checks of boundary and out of boundary values for GetVD and OnVD RCPs